apps/prairielearn/src/components/Navbar.html.ts

```diff
@@ -46,9 +46,13 @@ export function Navbar({
         `
       : ''}
 
-    <div class="container-fluid bg-primary">
-      <a href="#content" class="visually-hidden-focusable d-inline-flex p-2 m-2 text-white">
-        Skip to main content
+    <div class="container-fluid bg-primary visually-hidden-focusable">
+      <a href="#content" class="d-inline-flex p-2 m-2 text-white">Skip to main content</a>
+      <a
+        href="https://prairielearn.readthedocs.io/en/latest/student-guide/accessibility/"
+        class="d-inline-flex p-2 m-2 text-white"
+      >
```

Comment:
A few musings:

- What we're linking to is the student accessibility guide, but we're linking to it from all pages, including instructor ones. We could try to limit this only to student pages? Not sure if that's worth it. Alternatively, we could rebrand the student page as "user accessibility guide", and differentiate it from the "instructor accessibility guide" that we want to build.
- This _may_ be inaccessible in exam mode, but it might not be (e.g. BYOD, less-restrictive CBTFs, etc). I opted to show it unconditionally rather than hiding it in exam mode.

Reply:
I like the idea of making it a generic user accessibility guide, which could itself link to the instructor guide.

I agree that unconditional linking seems fine at least for now.

Reply:
I opened https://github.com/PrairieLearn/PrairieLearn/pull/11922 to add instructor-facing accesibility documentation. I can worry about how to integrate the two in the future 😅